src/scripts/main.js

```diff
@@ -1,3 +1,21 @@
 'use strict';
 
 // write your code here
+const population = [...document.querySelectorAll('span.population')];
+const numbers = population.map((e) => +e.textContent.split(',').join(''));
+
+const totalFunc = (nums) => {
+  let result = 0;
+
+  for (const num of nums) {
+    result += num;
+  }
+
+  return result;
+};
+
+const total = totalFunc(numbers).toLocaleString('en-US');
+const avarage = (totalFunc(numbers) / numbers.length).toLocaleString('en-US');
```

Comment:
There is a typo in the variable name 'avarage'. It should be 'average' to correctly reflect its purpose.